### PR TITLE
Implement ScenarioContext and split pipeline into jobs

### LINKED ISSUE #5: investigate if is it needed to provide custom store

### PR DESCRIPTION
**Pull request by paveliam**
resolves #5

### CODE REVIEW

File: aquality-selenium-template-cucumber/src/test/java/aquality/selenium/template/cucumber/features/Demo.feature [hunk 1]
```diff
@@ -12,4 +12,10 @@ Feature: Demo
...
     And I accept Privacy and Cookies Policy
     And I click Send button
-    Then Notification about empty fields is present
\ No newline at end of file
+    Then Notification about empty fields is present
+
+  @demo
+  Scenario: ScenarioContext demo
+    Given I store '2' as 'value1'
+    And I store '3' as 'value2'
```
Comment (mialeska) [inline aquality-selenium-template-cucumber/src/test/java/aquality/selenium/template/cucumber/features/Demo.feature:20]: "I store '2' as 'value1'" looks like a When step. The proper time of "Given" keyword is present perfect or the verb in passive voice, smth like `Given The value '2' is stored as 'value1'` or `I stored '2' as 'value1'`
Also we should add tabulation before `And` step.
And regarding `Then` step, it should define expected state, e.g. `Then The result of adding 'value1' to 'value2' is '5'`

File: aquality-selenium-template-cucumber/src/test/java/aquality/selenium/template/cucumber/stepdefinitions/ContactUsPageSteps.java [hunk 2]
```diff
@@ -6,12 +6,15 @@
...
 
+import javax.inject.Inject;
+
 import static org.testng.Assert.assertTrue;
 
 public class ContactUsPageSteps {
 
     private final ContactUsPage contactUsPage;
 
+    @Inject
```
Comment (mialeska) [inline aquality-selenium-template-cucumber/src/test/java/aquality/selenium/template/cucumber/stepdefinitions/ContactUsPageSteps.java]: what do you inject here?

**Review by mialeska — CHANGES_REQUESTED** [review]

File: aquality-selenium-template-cucumber/src/test/java/aquality/selenium/template/cucumber/stepdefinitions/ContactUsPageSteps.java (hunk 2 above)
Comment (paveliam) [reply aquality-selenium-template-cucumber/src/test/java/aquality/selenium/template/cucumber/stepdefinitions/ContactUsPageSteps.java]: nothing

File: aquality-selenium-template-cucumber/src/test/java/aquality/selenium/template/cucumber/features/Demo.feature (hunk 1 above)
Comment (paveliam) [reply aquality-selenium-template-cucumber/src/test/java/aquality/selenium/template/cucumber/features/Demo.feature:20]: It is just an example

**Review by mialeska — APPROVED** [review]

**Review by mialeska — APPROVED** [review]

**Pull request merged**